### ISSUE TITLE
Make sure the search algorithm searches on the README as well

### ISSUE THREAD

**Issue by addihorowitz** [cdklabs/construct-hub-webapp#254]
The construct hub returns less results than the the catalog. When we search for softchef we see only 4 packages while the awscdk.io returns 9 packages.

Reproduce: https://constructs.dev/search?q=softchef&offset=0

**Comment by iliapolo** [comment]
Its true that `awscdkio` produces 4 more packages than `constructs.dev` for `softchef` search query. The missing packages are:

- https://www.npmjs.com/package/sccdk-vue/v0.0.12
- https://www.npmjs.com/package/sccdk-vue/v1.0.10
- https://www.npmjs.com/package/sccdk-restapi/v/0.0.8
- https://www.npmjs.com/package/cdk-iot-security/v/0.0.0

However, the reason they don't appear in `constructs.dev` is not related to the search, these packages are actually missing from the `catalog.json` altogether. 

Sifting through the logs, it seems that we simply didn't get a change record for these packages from `npmjs.com`. I'd venture that this is a consequence of these packages being **delisted** from `npmjs.com` before we deployed `constructs.dev` - note that all the above links result in a 404. 

In addition, I verified that `awscdkio` **does not** perform a README search, nor do I think it or `constructs.dev` should..it has the potential for offering way more packages than the user intended. Given that this isn't the root cause of the issue described here with `softchef` packages, I think we should re-evaluate this feature.
